<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
font: 75 14pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string> screenTool</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string>连接设备：</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>591</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reconnect_btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重新检测设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnect_btn">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>断开连接设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addConfig_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_conn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Aharoni&quot;;</string>
   </property>
   <property name="text">
    <string>未连接服务器</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>118</width>
      <height>78</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
